commands.txt

```diff
@@ -7,10 +7,10 @@ SONAR_TOKEN=`$SD_CURL_CMD_WRAPPER "curl -H \"Authorization: Bearer $SD_TOKEN\" $
 VERSION=`cat sonar-project.properties | awk -F= '/sonar.projectVersion=/{print $2}'`
 PROJECT_VERSION=${VERSION:='not provided'}
 SONAR_OPTIONS=`eval echo "$SD_SONAR_OPTS"`
-if [ "$SD_SONAR_ENTERPRISE" = false ]; then SD_SONAR_PROJECT_KEY=`job:$SD_JOB_ID`; else SD_SONAR_PROJECT_KEY=`pipeline:$SD_PIPELINE_ID`; fi
-export SD_SONAR_PROJECT_KEY
+if [ "$SD_SONAR_ENTERPRISE" = false ]; then SD_SONAR_PROJECT_KEY="job:$SD_JOB_ID" && SD_SONAR_PROJECT_NAME="$SD_PIPELINE_NAME:$SD_JOB_NAME"; else SD_SONAR_PROJECT_KEY="pipeline:$SD_PIPELINE_ID" && SD_SONAR_PROJECT_NAME="$SD_PIPELINE_NAME"; fi
+export SD_SONAR_PROJECT_NAME
```

Comment:
do we need to compute this ? won't this already be available in build ? because of the call launcher makes https://github.com/screwdriver-cd/launcher/blob/master/launch.go#L529-L536

Reply:
removed